internal/backend/k8s.go

```diff
@@ -212,3 +217,31 @@ func (c *K8sPersister) ListAccessBindings(ctx context.Context, roleName, namespa
 	}
 	return list, nil
 }
+
+func (c *K8sPersister) GetApplication(ctx context.Context, name, namespace string) (*unstructured.Unstructured, error) {
```

Comment:
How about changing this function name to `GetApplicationUnstructured` to make it explicit that this type is required? It might confuse ppl as it uses a typed client to return an unstructured object. Consider explaining it in the godoc as well.

Reply:
Added in Godoc. If there is a need to have a function in the future that returns the typed object, then `GetApplication` and `GetApplicationUnstructured` make sense. I don't see the need for now. The return type makes it explicit.